apis/pkg/v1/interfaces.go

```diff
@@ -396,6 +396,12 @@ type PackageRevision interface {
 
 	GetESSTLSSecretName() *string
 	SetESSTLSSecretName(s *string)
```

Comment:
Which (if any) of the existing `ESSTLSSecretName` and `WebhookTLSSecretName` methods will be removed in a future PR? I think we plan to remove the webhook ones, but keep the ESS ones, right? If so it would help to add a comment explaining why we have client, server, and ESS specifically.

Reply:
I was planning to remove ESS as well. We can start using client certificates per entity and have a single server cert for ESS plugins. Our issue is that we're not deploying ESS plugins via package manager. Therefore, cannot generate different server certificates for them.

Reply:
Got it, thanks. I'd suggest adding comments noting they were deprecated but I think you're planning to fast follow to remove them right?

Reply:
Yes, that's what I was thinking. I'm also okay with labeling them as deprecated. But I don't think it will be noticed. 

However, we need to communicate that the providers' webhook and ESS certificate paths and their content will change after we start using the new ones.